src/content/api/merchants.mdx

```diff
@@ -58,6 +58,10 @@ which define the payment methods available for a Payment Request.
     Merchant [Settlement Config](#settlement-config-model).
   </Property>
 
+  <Property name="refundConfig" type="object">
```

Comment:
Should we add the experimental tag as this will likely be a model that changes over time?

Reply:
If it changes over time that's fine as long as we don't remove the bank account id field or break the API contract we setup. I think we over use the experimental flag.